src/network/network_server.cpp

```diff
@@ -1441,6 +1444,11 @@ NetworkRecvStatus ServerNetworkGameSocketHandler::Receive_CLIENT_SET_NAME(Packet
 	if (this->HasClientQuit()) return NETWORK_RECV_STATUS_CONN_LOST;
 
 	if (ci != nullptr) {
+		if (!NetworkIsValidClientName(client_name)) {
+			/* An invalid client name was given. This was already validated by the client. */
```

Comment:
"This was already validated by the client" <- what do you mean?

Reply:
I've changed the comment to be less terse:
An invalid client name was given. However, the client ensures the name is valid before it is sent over the network, so something went horribly wrong. This is probably someone trying to troll us.